knex.js

```diff
@@ -2,7 +2,7 @@ const knex = require("knex");
 const connectedKnex = knex({
     client: "sqlite3",
     connection: {
-        filename:"votes02.db"
+        filename:"./db/votes.db"
```

Comment:
I think this should come from an env var.

Reply:
Later 👍